tests/models/kosmos2/test_modeling_kosmos2.py

```diff
@@ -304,6 +304,25 @@ def test_forward_signature(self):
             expected_arg_names = ["pixel_values"]
             self.assertListEqual(arg_names[:1], expected_arg_names)
 
+    def test_load_save_without_tied_weights(self):
```

Comment:
`Kosmos-2` is the only model that requires this test overridden from the common tests: maybe I am doing something wrong when adding it.

Is this to be temporary here. I can take a look for this one later after PR being merged.

Reply:
It's because Kosmos has a `config.text_config` rather than just a `config`